spec/ddtrace/transport_spec.rb

```diff
@@ -0,0 +1,127 @@
+require 'spec_helper'
+
+require 'ddtrace'
+
+RSpec.describe Datadog::HTTPTransport do
+  let(:transport) { described_class.new('localhost', '8126', options) }
+  let(:options) { {} }
+
+  before(:each) do
+    @original_level = Datadog::Tracer.log.level
+    Datadog::Tracer.log.level = Logger::FATAL
+  end
+
+  after(:each) do
+    Datadog::Tracer.log.level = @original_level
+  end
+
+  describe '#handle_response' do
+    subject(:result) { transport.handle_response(response) }
+
+    context 'given an OK response' do
+      let(:response) { Net::HTTPResponse.new(1.0, 200, 'OK') }
+      it { is_expected.to be 200 }
+    end
+
+    context 'given nil' do
+      let(:response) { nil }
+      it { is_expected.to be 500 }
+    end
+  end
+
+  describe '#send' do
+    before(:each) { skip "TEST_DATADOG_INTEGRATION not set." unless ENV['TEST_DATADOG_INTEGRATION'] }
```

Comment:
perhaps we should use `rspec` tags for that?

Reply:
I don't follow?